core/src/main/java/org/mobilitydata/gtfsvalidator/notice/Notice.java

```diff
@@ -71,6 +71,6 @@ public String toString() {
 
   @Override
   public int hashCode() {
-    return Objects.hash(getCode(), getContext(), getSeverityLevel());
+    return Objects.hash(getClass().getSimpleName(), getContext(), getSeverityLevel());
```

Comment:
Objects.hash(getClass(), getContext(), getSeverityLevel());

I do not want to support use cases when you have classes with the same simple name but in different packages.

Reply:
Modified in https://github.com/MobilityData/gtfs-validator/pull/789/commits/c0732911e4b070236fb67dd0ded790698de764e8.